projects/postgresql/fuzzer/Makefile

```diff
@@ -40,18 +40,19 @@ fuzzer: simple_query_fuzzer \
 	json_parser_fuzzer
 
 
-json_parser_fuzzer: json_parser_fuzzer.o $(OBJS_FUZZERS)
-	$(CXX) $(CFLAGS) $(call expand_subsys,$^) -o $@ $(LIB_FUZZING_ENGINE)
-
 simple_query_fuzzer: simple_query_fuzzer.o $(OBJS_FUZZERS)
```

Comment:
you can actually make that shorter:

see e.g. https://makefiletutorial.com/ Multiple targets (Section 4.8)

IIUC (i am not very good with makefiles) you would then have just two rules:

simple_query_fuzzer.o json_parser_fuzzer.o: %.o: %.c
  $(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $^ 

simple_query_fuzzer json_parser_fuzzer: %: %.o $(OBJS_FUZZERS)
  $(CXX) $(CFLAGS) $(call expand_subsys,$^) -o $@ $(LIB_FUZZING_ENGINE)